models/clm/CAM_bias_correction/CAM6_bias_correct.m

```diff
@@ -0,0 +1,650 @@
+    sw_1hr=mean(reshape(sw_30,2,numel(sw_30)/2),1);        
+    lw_1hr=mean(reshape(lw_30,2,numel(lw_30)/2),1);        
+    ppt_1hr=mean(reshape(ppt_30,2,numel(ppt_30)/2),1);     
+    ps_1hr=mean(reshape(ps_30,2,numel(ps_30)/2),1);
+
+
+    ta_3hr=mean(reshape(ta_1hr,3,numel(ta_1hr)/3),1);       
+    q_3hr=mean(reshape(q_1hr,3,numel(q_1hr)/3),1);         
+    wind_3hr=mean(reshape(wind_1hr,3,numel(wind_1hr)/3),1); 
+    sw_3hr=mean(reshape(sw_1hr,3,numel(sw_1hr)/3),1);       
+    lw_3hr=mean(reshape(lw_1hr,3,numel(lw_1hr)/3),1);       
+    ppt_3hr=mean(reshape(ppt_1hr,3,numel(ppt_1hr)/3),1);    
+    ps_3hr=mean(reshape(ps_1hr,3,numel(ps_1hr)/3),1);       
+
+    clear ta_30 q_30 wind_30 sw_30 lw_30 ppt_30 ps_30
+
+
+      % Load CAM Ensemble Loop 
+      for jj = 1:ens_mem;  
+
+         %Downloading all ensemble members for all  reanalysis variables
+    
+         %SOLAR
+         Faxa_swndr(jj,:) = load_CAM('a2x1hi_Faxa_swndr',enstr{jj},yearstr{ii},path_CAM,'SOLAR');
+         Faxa_swvdr(jj,:) = load_CAM('a2x1hi_Faxa_swvdr',enstr{jj},yearstr{ii},path_CAM,'SOLAR');
+         Faxa_swndf(jj,:) = load_CAM('a2x1hi_Faxa_swndf',enstr{jj},yearstr{ii},path_CAM,'SOLAR');
+         Faxa_swvdf(jj,:) = load_CAM('a2x1hi_Faxa_swvdf',enstr{jj},yearstr{ii},path_CAM,'SOLAR');         
+
+         %NONSOLAR (3hr)
+         Faxa_rainc(jj,:) = load_CAM('a2x3h_Faxa_rainc',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Faxa_rainl(jj,:) = load_CAM('a2x3h_Faxa_rainl',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Faxa_snowc(jj,:) = load_CAM('a2x3h_Faxa_snowc',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Faxa_snowl(jj,:) = load_CAM('a2x3h_Faxa_snowl',enstr{jj},yearstr{ii},path_CAM,'hour3');
+
+         % 1hr STATE
+         Sa_u(jj,:) = load_CAM('a2x1h_Sa_u',enstr{jj},yearstr{ii},path_CAM,'hour1');
+         Sa_v(jj,:) = load_CAM('a2x1h_Sa_v',enstr{jj},yearstr{ii},path_CAM,'hour1');
+
+         % 3hr STATE
+         Sa_tbot(jj,:) = load_CAM('a2x3h_Sa_tbot',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Sa_shum(jj,:) = load_CAM('a2x3h_Sa_shum',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Sa_pbot(jj,:) = load_CAM('a2x3h_Sa_pbot',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Faxa_lwdn(jj,:) = load_CAM('a2x3h_Faxa_lwdn',enstr{jj},yearstr{ii},path_CAM,'hour3');
+
+         % 3hr STATE Unchanged
+         Sa_z(jj,:) = load_CAM('a2x3h_Sa_z',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Sa_ptem(jj,:) = load_CAM('a2x3h_Sa_ptem',enstr{jj},yearstr{ii},path_CAM,'hour3');     
+         Sa_dens(jj,:) = load_CAM('a2x3h_Sa_dens',enstr{jj},yearstr{ii},path_CAM,'hour3'); 
+         Sa_pslv(jj,:) = load_CAM('a2x3h_Sa_pslv',enstr{jj},yearstr{ii},path_CAM,'hour3');
+         Sa_topo(jj,:) = load_CAM('a2x3h_Sa_topo',enstr{jj},yearstr{ii},path_CAM,'hour3');
+
+      end %% Load CAM ensemble loop
+     
+  %% Initialize the ensemble mean scale values to the default of '0' which means no change

```

Comment:
This is still within the main loop, so it would be helpful to use consistent indenting to show that.